library/init/meta/expr.lean

```diff
@@ -32,6 +41,9 @@ meta constant expr.macro_def_name (d : macro_def) : name
 meta def expr.mk_var (n : nat) : expr :=
 expr.var (unsigned.of_nat n)
 
+meta constant expr.mk_quote_macro : expr → expr
+meta constant expr.mk_prenum_macro : nat → expr
+meta constant expr.mk_string_macro : string → expr
```

Comment:
@leodemoura Should I move all of these to `pexpr`? It's a bit weird since we also have `expr.mk_macro`.